app/api/helpers/files.py

```diff
@@ -196,6 +196,59 @@ def create_save_image_sizes(image_file, image_sizes_type, unique_identifier=None
     return new_images
 
 
+def create_system_image(image_file=None, upload_path=None, unique_identifier=None,
+                        ext='jpg', remove_after_upload=False):
+    """
+    Create System Images for Event Topics
+    :param upload_path:
+    :param ext:
+    :param remove_after_upload:
+    :param image_file:
+    :return:
+    """
+    # Get an unique identifier from uuid if not provided
+    filename = '{filename}.{ext}'.format(filename=get_file_name(), ext=ext)
+    if image_file:
+        data = urllib.request.urlopen(image_file).read()
```

Comment:
I think
```
with open urllib.request.urlopen(image_file) as img_data:
        image_file = io.BytesIO(img_data.read())
```
will be better as it will reduce the work of the garbage collector. Also, it's a recommended best-practice idiom.